website/docs/reference/resource-properties/description.md

```diff
@@ -357,7 +357,7 @@ version: 2
 
 models:
   - name: customers
-    description: "!\[dbt Logo](https://raw.githubusercontent.com/dbt-labs/dbt/develop/etc/dbt-logo.svg)"
+    description: "!\[dbt Logo](https://raw.githubusercontent.com/dbt-labs/dbt-core/develop/etc/dbt-logo.svg)"
```

Comment:
Both of these links 404 for me. The `raw.githubusercontent.com` means it was created from an image in a PR. Do we want to link to this image? https://github.com/dbt-labs/dbt-core/blob/main/etc/dbt-core.svg

Reply:
swapped